todo/src/style/index.css

```diff
@@ -0,0 +1,36 @@
+.bg-High {
```

Comment:
why do you use upper case in naming ?

Reply:
I change selector names.
```suggestion
.bg-high {
        background-color: var(--bg-light-high-priority);
    }
```